PD-1967 / 25.10 / Pd 1967 25 10 rc 1 release notes (#4143)
truenas/documentation: release-summary/TrueNAS-SCALE-25.10-RC.1.release-notes.txt

## TrueNAS 25.10-RC.1

===This a RC release of software available to early adopters who wish to evaluate new features===

This release candidate has software component updates and fixes for issues discovered in 25.10-BETA.1.

- **Breaking Changes:**
  - Nvidia open source driver supports the latest GPUs, but removes support for legacy architectures.
  - Removed idmap_autorid option from the Credentials/Directory Services/IDMAP options.
  - SMB share configuration now displays only options relevant to each preset; shares using "No Preset" migrate to "Legacy Share".
- **New Features:** Enhanced remote syslog configuration supporting up to two servers with individual transport and TLS options.
- **Improvements:** Password manager compatibility, NVMeoF target configuration, VM configuration performance.
- **Critical Bug Fixes:** Resolved upgrade blocking issues, SMB service hanging with Windows clients, and incorrect disk temperature alerts.

**Pre-Update Actions:**

>   - Verify installed Nvidia GPU is [supported by the open GPU kernel drivers](https://github.com/NVIDIA/open-gpu-kernel-modules/tree/570?tab=readme-ov-file#compatible-gpus) (Turing and later architectures supported; Pascal, Maxwell, and Volta architectures no longer supported).

Reviewer: pare down a bit
```suggestion
  - Verify installed Nvidia GPU is [supported](https://github.com/NVIDIA/open-gpu-kernel-modules/tree/570?tab=readme-ov-file#compatible-gpus) (Turing and later are supported; Pascal, Maxwell, and Volta are no longer supported).
```
